### DISCUSSION TITLE
Broken link in lesson 2: Feedback mechanisms & Information flow

### DISCUSSION THREAD

**Discussion by alcazar90** [OpenMined/courses#188] [Feedback] [answered]
Hi,

There is a broken link the additional resources on lesson 2: "Exposed the tricks sellers use to post fake reviews on Amazon".

https://www.which.co.uk/news/2019/07/exposed-the-tricks-sellers-use-to-post-fake-reviews-on-amazon/%20-%20Which?

best,
Cristóbal

**Comment by yemikifouly** [comment, marked as answer]
Hello Cristóbal! 👋 Thanks for bringing this up! I am trying to track this down. 🙂 Could you please share the concept of Lesson 2 that this broken link is in?

**Reply by alcazar90** [reply]
Hello @yemikifouly!

Of course, here is the path Course 1, Lesson 2, Concept: Feedback mechanisms & information flows.
As well the url:
https://courses.openmined.org/courses/our-privacy-opportunity/1f34d3a8-2fa1-4a1e-9ef1-eaaf9dddd617/04cea8cb-aa0d-4e95-bbb5-a20d768e9546

**Reply by yemikifouly** [reply]
Thanks Cristóbal! 🙌 Made it into [an issue](https://github.com/OpenMined/courses/issues/191). Thank you for bringing it to our attention. 🙂

**Reply by em-blue** [reply]
Fixed it! :) 
https://www.which.co.uk/news/2019/07/exposed-the-tricks-sellers-use-to-post-fake-reviews-on-amazon
